test/e2e/common_test.go

```diff
@@ -756,6 +756,13 @@ func SkipIfNotFedora() {
 	}
 }
 
+func SkipIfJournaldUnavailable() {
+	cmd := exec.Command("journalctl", "-n", "1")
+	if err := cmd.Run(); err != nil {
+		Skip("journald is unavailable")
+	}
```

Comment:
place cache this, i.e. use `sync.Once` and then store the result as bool and then just do a `if bool { Skip() }`